eth/_utils/numeric.py

```diff
@@ -125,3 +125,25 @@ def integer_squareroot(value: int) -> int:
         x = y
         y = (x + value // x) // 2
     return x
+
+
+def bitwise_xor(a: Hash32, b: Hash32) -> Hash32:
+    """
+    Return the xor of hash ``a`` and hash ``b``
+    """
+    if len(a) != 32:
```

Comment:
Depending on the performance needs for this function we could drop these two validation checks since the `Hash32` type checks give us *some* assurances..  I'll leave it to @hwwhww to decide if she thinks this is necessary/beneficial.

Reply:
It's "thoughtful" to have the checks for "utility function", but we definitely don't need it for RANDAO use case. Remove +1.